spiped/dispatch.c

```diff
@@ -95,13 +105,31 @@ doaccept(struct accept_state * A)
 
 /* A connection has closed.  Accept more if necessary. */
 static int
-callback_conndied(void * cookie)
+callback_conndied(void * cookie_node)
```

Comment:
Yeah, but it's a different cookie than all the other `callback_*()` functions in that `.c` file...?   Principle of least surprise?

*shrug*
ok, I'll un-change it.